src/lib/concurrency/transaction_manager.cpp

```diff
@@ -16,12 +16,12 @@ TransactionManager& TransactionManager::get() {
 void TransactionManager::reset() {
   auto& manager = get();
   manager._next_transaction_id = 1u;
-  manager._last_commit_id = 0u;
-  manager._last_commit_context = std::make_shared<CommitContext>(0u);
+  manager._last_commit_id = 1u;
+  manager._last_commit_context = std::make_shared<CommitContext>(1u);
```

Comment:
Should we replace these numbers with constants? `FIRST_COMMIT_ID`, `INVALID_COMMIT_ID`? Everything else looks good.

Reply:
Yes, we should. Done.